routes/api/contacts.js

```diff
@@ -1,25 +1,82 @@
-const express = require('express')
+const express = require('express');
+const { HttpError } = require('../../helpers');
+const Joi = require ('joi');
 
 const router = express.Router()
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const contacts = require("../../models/contacts");
+
+const addSchema = Joi.object({
+  name: Joi.string().required(),
+  email: Joi.string().required(),
+  phone: Joi.string().required(),
+});
+
+router.get('/',
+  async (req, res, next) => {
+    try {
+        res.json(await contacts.listContacts());
+    } catch (error) {
+      next(error);
+    }
+  }
+)
 
 router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+  try {
+    const { contactId } = req.params;
+    const response = await contacts.getContactById(contactId);
+    if (!response) {
+      throw HttpError(404,"Not found");
+      }
+      return res.json(response);
+  } catch (error) {
+      next(error);
+    }})
 
 router.post('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+  try {
```

Comment:
routes should only invoke function, and NOT define it. Separate functions, place then in controller file